server/logger.py

```diff
@@ -0,0 +1,26 @@
+import time
+import logging
+
+def Logger(name):
```

Comment:
Also, perhaps rename it to get_logger? If only to match the naming scheme of other functions?